python/ray/serve/backend_worker.py

```diff
@@ -274,15 +294,12 @@ async def invoke_batch(self, request_item_list: List[Query]) -> List[Any]:
                 raise RayServeException(error_message)
         except Exception as e:
             wrapped_exception = wrap_to_ray_error(e)
-            self.error_counter.record(1, {"backend": self.backend_tag})
+            self.error_counter.record(1)
             result_list = [wrapped_exception for _ in range(batch_size)]
 
-        self.processing_latency_tracker.record(
-            (time.time() - timing_start) * 1000, {
-                "backend": self.backend_tag,
-                "replica_tag": self.replica_tag,
-                "batch_size": str(batch_size)
-            })
+        self.processing_latency_tracker.with_tags({
```

Comment:
If we have the `with_tags` modifier, do we need the `default_tags` argument to the constructor at all? Couldn't I do something like this instead?
```
self.processing_latency_tracker = metrics.Histogram(...).with_tags(default_tags)
```